src/api_v2/node.md

```diff
@@ -90,6 +90,83 @@ HTTP/1.1 200 OK
     
      */
 
+== [GET] api/nodes 
+
+    /**
+    @api {get} /api/nodes 1. List accepted Nodes
+    @apiVersion 6.0.0
+    @apiName listAcceptedNodes
+    @apiGroup Nodes
+
+    @apiDescription That function allow you to get details of accepted Nodes. You can make query on those Nodes based on inventory criterion. You can use the same queries than those sued in NodeGroup queries
+
+    @apiParam (Mono) {Query}      query A query json object like we use as query in NodeGroups. A three field json object { 'composition' : 'and', 'select' : 'node', 'where' : [{"objectType":"node","attribute":"OS","comparator":"eq","value":"Linux"}]}, That parameter can be replace by the three following (select, composition, where) 
+    @apiParam (Mono) {String}          [composition=and] Boolean operator to use between each criteria. Other value is "And"
+    @apiParam (Mono) {String}          [select=node] What kid of data we wnt to include. Here we can get policy servers/relay by setting 'nodeAndPolicyServer'
+    @apiParam (Mono) {Criterion} where The criterion you want to find for your nodes like '[{"objectType":"node","attribute":"OS","comparator":"eq","value":"Linux"}]}'
+    @apiParam (Multi) {String}         [include=default] Level of information to include from the node inventory. Some base levels are defined (minimal, default, full). you can add fields you want by adding them to the list, possible values are keys from json answer. 
+    
+    @apiExample Example usage:
+     # Get all nodes having a hostname strating with node1 and based on Linux
+     curl -H "X-API-Token: yourToken" https://rudder.example.com/rudder/api/latest/nodes -d where=[{"objectType":"node","attribute":"OS","comparator":"eq","value":"Linux"},{"objectType":"node","attribute":"nodeHostname","comparator":"regex","value":"node1.*"}]
+
```

Comment:
This line does not work well on all shell if copy/pasted as is. 
Could you add simple quote for url and query so that curl/shell does not complain on most case, like that:
 curl -H "X-API-Token: dTxvl4eL8p3YqvwefVbaJLdy8DyEt7Vw" -X POST 'http://..../api/latest/nodes' -d 'where=[{"objectType":"node","attribute":"OS","comparator":"eq","value":"Linux"},{"objectType":"node","attribute":"nodeHostname","comparator":"regex","value":"node1.*"}]'

?